admiral/admiral_macros.h

```diff
@@ -49,6 +49,11 @@
 #define WAYPOINT_DEBUG_MARKER_SIZE          [0.5,0.5]
 #define CQC_DEBUG_MARKER_SIZE               [0.5,0.5]
 #define CQC_DEBUG_MARKER                    "mil_triangle"
+#define COUNTER_DEBUG_MARKER_TYPES          ["b_inf", "b_motor_inf", "b_armor","total"]
```

Comment:
I think this is not necessary with the changes I suggested as `total` behaves different compared to the rest of the markers. Should use the already define group marker macro instead, with adding a special case for total where necessary.